### PR TITLE
SHARD-670 : Remove network access for past employees (genesis accounts and dev/multi-sig keys)

### PR DESCRIPTION
**Pull request by abdulazeem-tk4vr**
linear : https://linear.app/shm/issue/SHARD-670/remove-network-access-for-past-employees-genesis-accounts-and-devmulti

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -325,59 +325,55 @@ config = merge(
...
+          /* prettier-ignore */ '000aa90686097de101bb5fad9cc4af6ccf568b4612d8dd032497a8ac9ccba91f': DevSecurityLevel.High,
+          /* prettier-ignore */ '4347a51c55921f7ffdf00ebc84d0849598a59fc9eb244bcbf5a4e26abad0a005': DevSecurityLevel.High,
+          /* prettier-ignore */ '3dfb1794a88ad3c19b63b9ef2006d45f7c01acedd7795908457f1470f8d10d2f': DevSecurityLevel.High,
+          /* prettier-ignore */ 'b17be71f65ec9804404de1333a93132b83a1d614a3d14a78db5c7e3219e49524': DevSecurityLevel.High,
+          /* prettier-ignore */ '26c333f353c06766cf811ba97572409848c90fb66291d8ef91e25c4d1bf439c7': DevSecurityLevel.High,
+          /* prettier-ignore */ 'abb118e65bbd834d3f9c3135f72a3ed883b5c3b85c9e4a647b142f2824663e20': DevSecurityLevel.High,
           // always prefix with prettier ignore
         },
-        multisigKeys: {
+        multisigkeys: {
```
Comment (aniketdivekar) [inline src/config/index.ts]: any reason we changed the key name here?

File: src/config/genesis_accounts.json [hunk 2]
```diff
@@ -521,5 +394,26 @@
...
+  },
+  "0x4Ab88E00570B740c1C403034D05954584d6351d1": {
+    "wei": "1001000000000000000000"
+  },
+  "0xD815DA50966c19261B34Ffa3bE50A30A67D97456": {
+    "wei": "1001000000000000000000"
+  },
+  "0x8d404dd7bb94ba73ff7609c8ec01c01e54afae6d": {
+    "wei": "1001000000000000000000"
   }
```
Comment (justin-shardeum) [inline src/config/genesis_accounts.json:418]: Kenny has requested that we add one more for him (genesis account with 1001 SHM): 

`0xCB65445D84D15F703813a2829bD1FD836942c9B7`

**Review by justin-shardeum — CHANGES_REQUESTED** [review]
Please hold off on merging this until I give the go-ahead.

I have a couple of people who've messaged me asking me for a little bit more time.  I'll let you know once those conversations are fully resolved.

File: src/config/genesis_accounts.json (hunk 2 above)
Comment (abdulazeem-tk4vr) [reply src/config/genesis_accounts.json:418]: @aniketdivekar looping you in

File: src/setup/sync.ts [hunk 3]
```diff
@@ -1,7 +1,7 @@
 import { DevSecurityLevel, Shardus, ShardusTypes } from '@shardus/core'
 import { Account, Address } from '@ethereumjs/util'
 import config from '../config'
-import genesis from '../config/genesis.json'
+import genesis from '../config/genesis_accounts.json'
```
Comment (justin-shardeum) [inline src/setup/sync.ts:4]: Let's refrain from renaming any files at this point where it isn't strictly necessary.

**Review by justin-shardeum — CHANGES_REQUESTED** [review]

File: src/setup/sync.ts (hunk 3 above)
Comment (abdulazeem-tk4vr) [reply src/setup/sync.ts:4]: I was instructed by the team to rename the file but it is alright. Please inform me what the final changes needed are, if any.

**Pull request closed without merging**